cypress/integration/integration_spec.js

```diff
@@ -153,7 +153,10 @@ describe('user', () => {
             logIn(newUser.username, newUser.password);
 
             // Log out
-            cy.getByText('Log out').click();
+            cy.get('.user-nav__links')
```

Comment:
If possible add a data-testid to this element to avoid needing a CSS hook here

Reply:
Not possible (I tried this first) – those nav links are generated from a loop/array.

Reply:
We control how this is generated though, so we could definitely add a hook in _somewhere_ not a big deal for now though.